less/type.less

```diff
@@ -347,18 +347,17 @@ strong {
   .bg-variant(@gray-base);
 }
 .bg-focus {
-  background-color: #FFF;
+  background-color: @brand-white;
 }
 
-.jumbotron,
+
 .bg-primary {
   color: @brand-smoke-plus-20;
```

Comment:
@StevePole in between this line and :374, which one do we actually use (i'd want to remove the one that is outdated if possible)